src/SFA.DAS.ApprenticeCommitments/Application/Commands/SendInvitationReminders/SendInvitationRemindersCommand.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using MediatR;
+
+namespace SFA.DAS.ApprenticeCommitments.Application.Commands.SendInvitationReminders
+{
+    public class SendInvitationRemindersCommand : IRequest
+    {
+        public DateTime SendNow { set; get; }
+        public int RemindAfterDays { set; get; }
```

Comment:
I can't find where `RemindAfterDays` is used.  Should it be passed through to the inner API?

Reply:
oh dear, did I miss it from the query